vm/src/builtins/enumerate.rs

```diff
@@ -60,6 +61,52 @@ impl PyIter for PyEnumerate {
     }
 }
 
+#[pyclass(module = false, name = "reversed")]
+#[derive(Debug)]
+pub struct PyReverseSequenceIterator {
+    pub position: AtomicCell<isize>,
```

Comment:
is this isize rather than a usize?

Reply:
Usually the reverse iterators keep an `isize` in order to check that its `> 0` while decreasing it and indexing the underlying object.